src/app/core/services/scheduling.service.ts

```diff
@@ -125,49 +105,32 @@ export class SchedulingService {
   }
 
   getNonReportedCompletedTasks() {
-    const defaultTasks = this.getDefaultTasks()
-    const clinicalTasks = this.getClinicalTasks()
-    return Promise.resolve(
-      Promise.all([defaultTasks, clinicalTasks]).then(
-        defaultAndClinicalTasks => {
-          const tasks = defaultAndClinicalTasks[0].concat(
-            defaultAndClinicalTasks[1]
+    return Promise.all([this.getDefaultTasks(), this.getClinicalTasks()]).then(
+      defaultAndClinicalTasks => {
```

Comment:
i think if `([defaultTasks, clinicalTasks])` syntax can be used, it would make the code clearer and concise.